packages/eslint-plugin/src/rules/require-localize-metadata.ts

```diff
@@ -0,0 +1,87 @@
+import type { TSESTree } from '@typescript-eslint/experimental-utils';
+import { createESLintRule } from '../utils/create-eslint-rule';
+
+type Options = [
+  {
+    readonly requireDescription?: boolean;
+  },
+];
+
+const DEFAULT_OPTIONS: Options[number] = {
+  requireDescription: false,
+};
+
+const VALID_LOCALIZED_STRING_WITH_DESCRIPTION = new RegExp(/`:[^|]*:.*`/);
+
+// TODO: The following values are also used in eslint-plugin-template/i18n, and should likely be centralized
```

Comment:
What is a good location for these values?

Reply:
I would remove the comment, I think having the small bit of repetition is fine and allows for them to remain independent and be more easily refactored if the structure of Angular guides changes in future